src/views/activity/components/bridge-card/bridge-card.view.tsx

```diff
@@ -24,14 +30,78 @@ export interface BridgeCardProps {
 
 const BridgeCard: FC<BridgeCardProps> = ({
   bridge,
+  env,
   isFinaliseDisabled,
   networkError,
   onClaim,
   showFiatAmount,
 }) => {
-  const { amount, fiatAmount, status, to, token } = bridge;
+  const { amount, depositTxHash, fiatAmount, from, status, to, token } = bridge;
   const classes = useBridgeCardStyles();
   const navigate = useNavigate();
+  const [lastVerifiedBatch, setLastVerifiedBatch] = useState<AsyncTask<BigNumber, string>>({
+    status: "pending",
+  });
+  const [batchNumberOfL2Block, setBatchNumberOfL2Block] = useState<AsyncTask<BigNumber, string>>({
+    status: "pending",
+  });
+
+  useEffect(() => {
+    if (status === "initiated" && from.key === "polygon-zkevm") {
+      setBatchNumberOfL2Block({ status: "loading" });
+      getBatchNumberOfL2Block(env.chains[1].provider, depositTxHash)
```

Comment:
Instead of sending the `depositTxHash` to later retrieve the `block` in which the tx was included, why we don't just send the `block` returned by the API to the function and we just retrieve the batch?

<img width="618" alt="Captura de pantalla 2023-01-24 a las 12 37 14" src="https://user-images.githubusercontent.com/6737805/214282201-6969bdba-af32-4dde-bbec-f13c22a82ebb.png">

Reply:
Good catch! I did not remember that the block number was there. It wasn't even parsed yet.

Done!